### PR TITLE
Added the Logical and bitwise operators section, fixes merge issue with  #1817

### PR DESCRIPTION
**Pull request by jatindhankhar**
- [x] PR touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] YAML Frontmatter formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Seriously, look at it now. Watch for quotes and double-check field names.

### CODE REVIEW

**Comment by jatindhankhar** [comment]
Sorry for wasting your time with the previous PR.
Hopefully, Fixes merge issues with #1817

File: c++.html.markdown [hunk 1]
```diff
@@ -1078,6 +1079,47 @@ cout<<it->second;
...
 
+///////////////////////////////////
+// Logical and Bitwise operators
+//////////////////////////////////
+
+// Most of the operators in C++ are same as in other languages
+
+// Logical operators
+
+true && false // Performs **logical and** to yield false
```
Comment (vendethiel) [inline c++.html.markdown]: can you add a note about the fact they short-circuit?

Comment (jatindhankhar) [inline c++.html.markdown]: Done

**Comment by vendethiel** [comment]
Thanks!

**Pull request merged**